.github/workflows/github_actions.yml

```diff
@@ -154,7 +154,7 @@ jobs:
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        os: [ macos-11, ubuntu-latest, macos-14]
+        os: [ macos-13, ubuntu-latest, macos-14]
```

Comment:
Do we need both 13 and 14? C# does just `macos-latest` for example.

Reply:
macOS 14 is for Apple Silicon macs, macOS-13 intel